origami.py

```diff
@@ -750,6 +750,33 @@ def run(self):
         new_window = sublime.active_window()
         new_window.set_layout(layout)
 
+class NewWindowWithFavouriteLayoutCommand(PaneCommand, WithSettings):
+    """Opens a new window using the favourite layout settings."""
+
+    def __init__(self, window):
+        self.window = window
+        super(NewWindowWithFavouriteLayoutCommand, self).__init__(window)
+
+    def run(self):
+        if not self.settings().has('saved_layouts'):
+            return
+
+        saved_layouts = self.settings().get('saved_layouts')
+        favourite_layout = next(
+            (layout for layout in saved_layouts if layout['favourite']), None)
+
+        if favourite_layout:
+            layout = {
+                'cells': favourite_layout['cells'],
+                'cols': favourite_layout['cols'],
+                'rows': favourite_layout['rows']
+            }  # type: sublime.Layout
+
+            self.window.run_command("new_window")
+            new_window = sublime.active_window()
+            if new_window.is_valid():
+                new_window.set_layout(layout)
+
```

Comment:
You've introduced a concept of favorites but haven't added any documentation for the user on how to make or use those.

But anyway, I'm not sure this is the right approach. It feels unnecessarily complicated. If your issue is with the quick panel showing up unnecessarily then maybe we could just make it pick the saved layout automatically if there is just one.

Alternatively we could teach `new_window_from_saved_layout` command to accept a nickname so that you could make your own command that opens a layout with specific name.